zipline/finance/performance/period.py

```diff
@@ -87,6 +87,28 @@
 log = logbook.Logger('Performance')
 
 
+class FastSeries(object):
+    def __init__(self, *args, **kwargs):
+        super(FastSeries, self).__init__(*args, **kwargs)
```

Comment:
Why are we forwarding all arguments to `object.__init__`, this only raises warning on py3 and is a nop.